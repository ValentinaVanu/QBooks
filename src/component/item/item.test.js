import { Item } from '.'
import renderer from 'react-test-renderer'
import { Router } from '@reach/router'


test('renders main item component', () => {
  const item = renderer.create(<Router><Item path="/item/:itemId"/></Router>).toJSON()
  expect(item).toMatchSnapshot()
})
