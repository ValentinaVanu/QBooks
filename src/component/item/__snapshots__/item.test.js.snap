// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders main item component 1`] = `null`;
